website/source/intro/getting-started/cluster.html.md

```diff
@@ -105,6 +105,11 @@ client {
 ports {
     http = 5656
 }
+advertise {
+  http = "localhost"
+  rpc = "localhost"
+  serf = "localhost"
```

Comment:
All I know is the examples in the getting started guide didn't work. When I went to the repo mentioned in the getting started guide, those configs worked and they used localhost. 
https://github.com/hashicorp/nomad/blob/master/demo/vagrant/client1.hcl